Tests/DownpourTests/DownpourTests.swift

```diff
@@ -0,0 +1,157 @@
+//
+//  DownpourTests.swift
+//  Downpour
+//
+//  Created by Kyle Fuller on 25/12/2016.
+//  Copyright © 2016 Stephen Radford. All rights reserved.
+//
+
+import XCTest
+import Downpour
+
+
+class DownpourTests: XCTestCase {
+    func testStandardShow1() {
+        let downpour = Downpour(string: "Mr.Show.Name.S01E02.Source.Quality.Etc-Group")
+        XCTAssertEqual(downpour.title, "Mr Show Name")
+        XCTAssertEqual(downpour.season, "01")
+        XCTAssertEqual(downpour.episode, "02")
+        XCTAssertEqual(downpour.type, .Some(.TV))
+        XCTAssertNil(downpour.year)
+    }
+
+    func testStandardShow2() {
+        let downpour = Downpour(string: "Show.Name.S01E02")
+        XCTAssertEqual(downpour.title, "Show Name")
+        XCTAssertEqual(downpour.season, "01")
+        XCTAssertEqual(downpour.episode, "02")
+        XCTAssertEqual(downpour.type, .Some(.TV))
+        XCTAssertNil(downpour.year)
+    }
+
+    func testStandardShow3() {
+        let downpour = Downpour(string: "Show Name - S01E02 - My Ep Name")
+        XCTAssertEqual(downpour.title, "Show Name")
+        XCTAssertEqual(downpour.season, "01")
+        XCTAssertEqual(downpour.episode, "02")
+        XCTAssertEqual(downpour.type, .Some(.TV))
+        XCTAssertNil(downpour.year)
+    }
+
+    func xtestStandardShow4() {
```

Comment:
This test is purposely disabled since the current implementation doesn't match the test. Left it in so we could see to adding support some time. (Didn't want to get too far down the rabbit hole and fix this at the moment).

There are a couple of other FIXMEs when there is minor formatting issues in tests below.